recipe_scrapers/barefootinthepines.py

```diff
@@ -0,0 +1,31 @@
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+
+
+class BarefootInThePines(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "barefootinthepines.com"
+
+    def ingredient_groups(self):
+        return group_ingredients(
+            self.ingredients(),
+            self.soup,
+            ".mv-create-ingredients h4",
+            ".mv-create-ingredients ul li",
+        )
+
+    def nutrients(self):
+        nutrition_data = {}
+        nutrition_container = self.soup.select_one(".mv-create-nutrition-box")
+        if not nutrition_container:
+            return nutrition_data
+
+        for item in nutrition_container.select(".mv-create-nutrition-item"):
+            label = item.select_one(".mv-create-nutrition-label")
+            if label:
+                label_text = label.get_text(strip=True)
+                quantity = item.get_text(strip=True).replace(label_text, "").strip()
+                nutrition_data[label_text] = quantity
```

Comment:
I think we should probably try to map the `label_text` values here to the well-known [`schema.org` `NutritionInformation` field names](https://www.schema.org/NutritionInformation).

Reply:
I just pushed an update that matches them against their schema value, let me know what you think